src/main/java/com/expensetracker/service/impl/ExpenseServiceImpl.java

```diff
@@ -68,6 +69,16 @@ public List<Expense> getAllExpensesPeriodByDate(int month, int year) {
         return expenseRepository.getExpenseByExpenseDateBetween(startDate, endDate);
     }
 
+    @Override
+    public List<YearExpensesDTO> getAllExpensesByYear(int year) {
+        LocalDate localDate = LocalDate.now().withYear(year);
+        LocalDate startDate = localDate.with(TemporalAdjusters.firstDayOfYear());
+        LocalDate endDate = localDate.with(TemporalAdjusters.lastDayOfYear());
+
+        System.out.println(expenseRepository.getExpensesByYear(startDate, endDate));
```

Comment:
Wouldn't be better to use a logger?

Reply:
Added issue to add a logger and removed all the print methods in the backend code